component.json

```diff
@@ -0,0 +1,74 @@
+{
+  "title": "CSV",
+  "description": "A comma-separated values (CSV) file stores tabular data (numbers and text) in plain-text form",
+  "buildType": "docker",
+  "version": "1.0.0",
+  "actions": {
+    "read_action": {
+      "main": "./dist/actions/read.js",
+      "title": "Read CSV attachment",
+      "help": {
+        "description": "This action will read the CSV attachment of the incoming message or from the specified URL and output a JSON object."
+      },
+      "fields": {
```

Comment:
I don't know if it does yet, but if it uses the ferryman-extensions attachment processor there should be a setting for the attachment processor url as well